joeyHandoverOG/0D_Solver.py

```diff
@@ -0,0 +1,67 @@
+# 0-D Transport Solver for OpenMC Validation
+import scipy
+import numpy as np
+import pandas as pd
+from scipy.optimize import fsolve
+
+# Define Known Variables from OpenMC Output
+
+# Unit Conversion
+cmm = 100
+
+# Phi_1
+p1 = 1
+
+# Absorption
+absp = [0, 0, 0, 0, 0, 0]
+# df = pd.read_csv(r"\\wsl$\Ubuntu\home\jeickman\testbin\OpenMC\for_joey_8_22\cross_section_gen\mgxs\absorption-xs.csv")
+df = pd.read_csv("~/Desktop/ffh/scriptsOpenMC2GF/testing/MOSART_fuel/mgxs/absorption-xs.csv")
+i=0
+for row in df['mean']:
+    absp[i] = row*cmm
+    i += 1 
+print('absorption: ', absp)
+# Scattering
+sct = [[0, 0, 0, 0, 0, 0],[0, 0, 0, 0, 0, 0],[0, 0, 0, 0, 0, 0],[0, 0, 0, 0, 0, 0],[0, 0, 0, 0, 0, 0],[0, 0, 0, 0, 0, 0]]
+# df = pd.read_csv(r"\\wsl$\Ubuntu\home\jeickman\testbin\OpenMC\for_joey_8_22\cross_section_gen\mgxs\scatteringmatrix-xs.csv")
+df = pd.read_csv("~/Desktop/ffh/scriptsOpenMC2GF/testing/MOSART_fuel/mgxs/scatteringmatrix-xs.csv")

```

Comment:
maybe we have a path arugment? it's a little unrefined to have different commented out paths. could put the options in a readme if you wanna save old paths

Reply:
I'm not going to add too much to this script since it's a very general and simple script, but I agree that the old paths shouldn't be in the pushed/clean version. I changed them to be relative pathing